src/Runner.jl

```diff
@@ -743,13 +743,13 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
     function rustc(io::IO, p::AbstractPlatform)
         extra_cmds = """
         if [[ " \${ARGS[@]} " == *'--target'* ]]; then
-            if ! [[ " \${ARGS[@]} " =~ --target(=| )$(map_rust_target(p)) ]]; then
-                echo "Attempting to invoke targeted 'rustc' wrapper with a different target! (Expected $(map_rust_target(p)))" >&2
+            if ! [[ " \${ARGS[@]} " =~ --target(=| )\${CARGO_BUILD_TARGET} ]]; then
+                echo "Attempting to invoke targeted 'rustc' wrapper with a different target! (Expected \${CARGO_BUILD_TARGET}, which is CARGO_BUILD_TARGET)" >&2
```

Comment:
But I don't understand this message now.  It'd read something like

> (Expected x86_64-unknown-linux-gnu, which is CARGO_BUILD_TARGET)

Reply:
I wanted to say why it expects that. Would this be ttetr?
```suggestion
                echo "Attempting to invoke targeted 'rustc' wrapper with a different target! (Expected CARGO_BUILD_TARGET=\${CARGO_BUILD_TARGET})" >&2
```